main-app/src/main/java/com/uzum/main/rabbit/RabbitConf.java

```diff
@@ -0,0 +1,52 @@
+package com.uzum.main.rabbit;
+
+import org.springframework.amqp.core.*;
+import org.springframework.amqp.rabbit.connection.CachingConnectionFactory;
+import org.springframework.amqp.rabbit.connection.ConnectionFactory;
+import org.springframework.amqp.rabbit.core.RabbitAdmin;
+import org.springframework.amqp.rabbit.core.RabbitTemplate;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.core.env.Environment;
+
+@Configuration
+public class RabbitConf {
+
+    @Autowired
+    private Environment env;
+
+    @Bean
+    public ConnectionFactory connectionFactory() {
+        CachingConnectionFactory cachingConnectionFactory = new CachingConnectionFactory(env.getProperty("env.host"));
+        cachingConnectionFactory.setUsername(env.getProperty("env.username"));
```

Comment:
having two similar names "env" (variable and config group name) making it less readable